gtsfm/retriever/netvlad_retriever.py

```diff
@@ -0,0 +1,167 @@
+"""Retriever implementation which provides a NetVLAD global image descriptor to suggest potential image pairs.
+
+Note: Similarity computation based off of Paul-Edouard Sarlin's HLOC:
+Reference: https://github.com/cvg/Hierarchical-Localization/blob/master/hloc/pairs_from_retrieval.py
```

Comment:
Referencing the [NetVLAD paper](https://openaccess.thecvf.com/content_cvpr_2016/papers/Arandjelovic_NetVLAD_CNN_Architecture_CVPR_2016_paper.pdf) may be more appropriate here.

Reply:
Good point, I'll add it.